doc/ref_model/chapters/chapter07.md

```diff
@@ -413,8 +413,7 @@ Security certification should encompass the following elements:
 
 |  Ref | Requirement  | Definition/Note  |
 |-------|------|-------|
-| req.sec.sys.001 | The Platform **must** support authenticated and secure APIs, API endpoints. | |
-| | The Platform **must** implement authenticated and secure access to GUI. | |
+| req.sec.sys.001 | The Platform **must** support authenticated and secure access to API, GUI and command line interfaces. | |
```

Comment:
@karinesevilla If you agree with my suggestion, this can be cancelled leaving req.sec.sys.001 as currently in the document. Thanks

Reply:
@pgoyal01 Yes it was the proposal "The Platform must support authenticated and secure access to API, GUI and command line interfaces."
And keep it as it is in RA1